pkgs/development/libraries/libsciter/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, glib
+, cairo
+, libuuid
+, pango
+, gdk-pixbuf
+, gtk3
+, stdenv
+, fetchurl
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libsciter";
+  version = "4.4.8.23-bis";
+
+  src = fetchurl {
+    url = "https://github.com/c-smile/sciter-sdk/raw/9f1724a45f5a53c4d513b02ed01cdbdab08fa0e5/bin.lnx/x64/libsciter-gtk.so";
```

Comment:
Since we're using this commit instead of a stable release, `version` should be in the `unstable-<date>` format i think

Reply:
> Since we're using this commit instead of a stable release, `version` should be in the `unstable-<date>` format i think

The repository is only used to host binaries for each release, each new commit title states the version like so:
![image](https://user-images.githubusercontent.com/17183803/155205082-39262864-5a8b-43a6-bb7d-94213b592ed3.png)

I think it is fair to treat the commit titles as version tags, but if you think it's better setting it as unstable, i will change it.
(The commits with the same version title are due to the owner uploading the Linux and Windows version separately)

Reply:
The version makes sense than, although I think a comment explaining that would be nice. Just to make it a bit easier for people to update this package in the future